site/docs/_shared/data/plugins.ts

```diff
@@ -335,6 +335,20 @@ export const PLUGINS = [
     },
     vulnerabilityType: 'criminal',
   },
+  {
+    category: 'Compliance and Legal',
+    description: 'Content related to RAG Full Document Exfiltration',
+    label: 'harmful',
+    link: '/docs/red-team/plugins/harmful/',
+    name: 'RAG Full Document Exfiltration',
+    pluginId: 'harmful:rag-full-document-exfiltration',
```

Comment:
we should give this a different namespace. It doesn't really make sense in the context of harmful. Can we just do `rag-full-document-exfiltration`?